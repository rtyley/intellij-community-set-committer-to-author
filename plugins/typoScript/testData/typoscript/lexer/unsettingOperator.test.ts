 myObject >
object > some irrelevant text
nextObjext.secondProperty > [GLOBAL] {test=3} (value)