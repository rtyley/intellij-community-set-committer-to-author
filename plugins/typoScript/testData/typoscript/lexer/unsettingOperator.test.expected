WHITE_SPACE (' ')
object path entity ('myObject')
WHITE_SPACE (' ')
> ('>')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('object')
WHITE_SPACE (' ')
> ('>')
IGNORED_TEXT (' some irrelevant text')
WHITE_SPACE_WITH_NEW_LINE ('
')
object path entity ('nextObjext')
. ('.')
object path entity ('secondProperty')
WHITE_SPACE (' ')
> ('>')
IGNORED_TEXT (' [GLOBAL] {test=3} (value)')